pkg/api/v1alpha1/hostosconfig.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+func validateHostOSConfig(config *HostOSConfiguration, machineName string) error {
+	if config == nil {
+		return nil
+	}
+
+	if err := validateVSphereMachineConfigNTPServers(config, machineName); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateVSphereMachineConfigNTPServers(config *HostOSConfiguration, machineName string) error {
```

Comment:
This doesn't seem to have anything vSphere specific in it?

Reply:
fixed